portfolio/src/main/java/com/google/sps/data/SearchQuery.java

```diff
@@ -28,12 +28,12 @@
     // Make a query based on salience scores given text and a query size.
     public SearchQuery(Video video, int querySize) {
         String queryText;
+        List<String> keywordList;
         if (languageService == null) {
-            queryText = video.getTitle();
+            keywordList = Arrays.asList(video.getTitle().split(" "));
```

Comment:
Maybe split it on a regular expression of empty space.

Reply:
I did it this way so that when I joined with a space after refiltering, the string should be the exact same (if I filter with a regex this will not be the case). Is that fine?

Reply:
Nvm I see what you're saying.